### ISSUE TITLE
Add support for macOS

### ISSUE THREAD

**Issue by conruggles** [rugglcon/python-backgroundchanger#1]
If support for mac could be added, that would be awesome.

This would mean adding a command for changing the mac background, and potentially fixing the config file logic to work with mac paths.

**Comment by wevnasc** [comment]
I would love to handle that, could you assign it to me?

**Comment by conruggles** [comment]
Awesome, thanks!

**Comment by wevnasc** [comment]
Hey @rugglcon I was think to change a little bit the code to create a kind of pipeline, in this way I can work in each piece of code at time in little PRs instead of change everything in one shot. WDYT?

**Comment by conruggles** [comment]
Yeah go ahead, if you do it please use GitHub actions. you should be able to follow normal pypi uploading pipeline that GH probably has a template for, I don't need to do any special pre- or post-processing.

**Comment by conruggles** [comment]
@wevnasc I created #4 for doing that, go ahead and interact with that issue so I can assign you
